Update azure-stack-add-default-image.md

gridpatrik opened the pull request:
A "`" was missing before a line break in the ADFS script causing the script to prompt for EnvironmentName.

azurecla commented:
Hi __@doctordns__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tfitzmac commented:
doctordns, thanks for submitting this fix. I had made the same change in another pull request that should get published today. So, we are going to close this one, but the fix will appear soon.

gridpatrik committed:
Update azure-stack-add-default-image.md

A "`" was missing before a line break in the ADFS script causing the script to prompt for EnvironmentName.
